app/templates/players/index.hbs

```diff
@@ -25,4 +25,23 @@
     <button {{on "click" (fn this.onFilterClick)}}>Filter</button>
 </label>
 
-<Players::List @players={{model}} @onHeaderClick={{this.onHeaderClick}} @sortDirection={{this.sortDirection}} />
+<Players::List
+    @players={{model}}
+
+    @onHeaderClick={{this.onHeaderClick}}
+    @sortDirection={{this.sortDirection}}
+
+    @limitValues={{this.limitValues}}
+    @limit={{this.limit}}
+    @selectLimit={{this.selectLimit}}/>
+
+<div class="pages">
+    <p>Pages</p>
+
+    {{#each this.pageArray as |index|}}
+    <button class={{if (eq index this.page) "disabled"}} disabled={{if (eq index this.page) "disabled"}}
```

Comment:
we can apply style on button:disabled I guess, instead of checking it twice and applying a class